src/commands/CreateCommand.js

```diff
@@ -0,0 +1,33 @@
+import Command from "../Command";
+import ConfigUtilities from "../ConfigUtilities";
+import FileSystemUtilities from "../FileSystemUtilities";
+import NpmUtilities from "../NpmUtilities";
+import path from "path";
+
+export default class LsCommand extends Command {
+  initialize(callback) {
+    this.pkgName = this.input[0];
+
+    callback(null, true);
+  }
+
+  execute(callback) {
+    const author = NpmUtilities.getWhoIAm();
+    const name = this.pkgName;
+    const pkgRoot = path.join(".", "packages", this.pkgName);
+    const registry = ConfigUtilities.read(this.repository.rootPath).registry;
+    FileSystemUtilities.mkdirp(pkgRoot, () => {
+      const manifest = {
+        name,
+        author,
+        version: "1.0.0"
+      };
+      if (registry) {
+        manifest.publishConfig = { registry };
```

Comment:
Setting [publishConfig in package.json](https://docs.npmjs.com/files/package.json#publishconfig) is safer because you can forget to specify the correct registry when using npm or lerna on the command line.  I think we should probably _also_ specify it on the command line, so you catch the change in the registry immediately, but I would also like the added safety if someone uses npm directly.